apps/docs/components/blocks/Search.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: DefaultLayout
+hideBreadcrumbs: true
+description: Search is a simple yet powerful input that helps users quickly find products in your shop.
+hideToc: true
+---
+
+# Search
+
+{{ $frontmatter.description }}
+
+## Basic search
+
+Simple search with an autocomplete functionality. Give your users hints of what they may look for.
```

Comment:
chore: maybe we can emphasize that the autocomplete functionality is mocked and should be replaced with actual API response?

Reply:
Done [`2e943fa`](https://github.com/vuestorefront/storefront-ui/pull/2630/commits/2e943fa5937d9562836d3ec710b01e4936e1885b)